### ISSUE TITLE
Not able to use webpack as builder 

### ISSUE THREAD

**Issue by lokeshparmar998** [nuxt/nuxt#19141]
### Environment

- Operating System: `Darwin`
- Node Version:     `v16.19.1`
- Nuxt Version:     `3.2.2`
- Nitro Version:    `2.2.2`
- Package Manager:  `npm@8.19.3`
- Builder:          `webpack`
- User Config:      `builder`
- Runtime Modules:  `-`
- Build Modules:    `-`

### Reproduction

Follow the link for reproduction: 
https://stackblitz.com/edit/nuxt-starter-wnjajh?file=nuxt.config.ts,package.json

`"devDependencies": {
    "@nuxt/webpack-builder": "^3.2.2",
    "nuxt": "^3.2.2"
  }`
  
  nuxt.config.ts
  `export default defineNuxtConfig({
    builder: 'webpack'
})`

### Describe the bug

Cannot run dev server, getting this error on console:

Cannot start nuxt:  Named export 'RuntimeGlobals' not found. The requested module 'webpack' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'webpack';
const { RuntimeGlobals } = pkg;

  import webpack, { RuntimeGlobals } from 'webpack';
  ^^^^^^^^^^^^^^
  SyntaxError: Named export 'RuntimeGlobals' not found. The requested module 'webpack' is a CommonJS module, which may not support all module.exports as named exports.
  CommonJS modules can always be imported via the default export, for example using:
  
  import pkg from 'webpack';                                                                                                       19:17:15
  const { RuntimeGlobals } = pkg;
  
  at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
  at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)
  at async Promise.all (index 0)
  at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
  at async bundle (node_modules/nuxt/dist/index.mjs:2699:76)
  at async build (node_modules/nuxt/dist/index.mjs:2675:5)
  at async Promise.all (index 1)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6809:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6863:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:51:20)

### Logs

```shell-script
Cannot start nuxt:  Named export 'RuntimeGlobals' not found. The requested module 'webpack' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'webpack';
const { RuntimeGlobals } = pkg;


  import webpack, { RuntimeGlobals } from 'webpack';
  ^^^^^^^^^^^^^^
  SyntaxError: Named export 'RuntimeGlobals' not found. The requested module 'webpack' is a CommonJS module, which may not support all module.exports as named exports.
  CommonJS modules can always be imported via the default export, for example using:
  
  import pkg from 'webpack';                                                                                                       19:17:15
  const { RuntimeGlobals } = pkg;
  
  at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
  at async ModuleJob.run (node:internal/modules/esm/module_job:189:5)
  at async Promise.all (index 0)
  at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
  at async bundle (node_modules/nuxt/dist/index.mjs:2699:76)
  at async build (node_modules/nuxt/dist/index.mjs:2675:5)
  at async Promise.all (index 1)
  at async load (node_modules/nuxi/dist/chunks/dev.mjs:6809:9)
  at async Object.invoke (node_modules/nuxi/dist/chunks/dev.mjs:6863:5)
  at async _main (node_modules/nuxi/dist/cli.mjs:51:20)
```

**Comment by Abrar535** [comment]
facing the same issue, is there any workaround?

**Comment by danielroe** [comment]
For now you can pin the version of webpack builder to `3.2.0`.

**Comment by danielroe** [comment]
Apologies for the inconvenience. This will shortly be released in the [edge channel](https://nuxt.com/docs/guide/going-further/edge-channel/), and will be available in the `3.2.3` patch release.

**Comment by lokeshparmar998** [comment]
> Apologies for the inconvenience. This will shortly be released in the [edge channel](https://nuxt.com/docs/guide/going-further/edge-channel/), and will be available in the `3.2.3` patch release.

When will the `3.2.3` patch be released? Do you have any estimated date you could provide?

**Comment by danielroe** [comment]
Likely today or tomorrow.
